### DISCUSSION TITLE
Welcome to RBucket Discussions!

### DISCUSSION THREAD

**Discussion by egnimos** [RBucket-Org/rb-docs#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by egnimos** [comment]
hey guys!! I want to ask something please provide your feedback on that, I need some suggestions, do write your reasons also, Thankuu

**Here are some features List that, I'am thinking about to add it, Please Suggest me on that, Whether to add the feature to RBucket right now** 

1. Transfer Bucket - it will transfer the bucket from one user to other user 
2. Groupee - it will group the post into single category, and you can publish it to the timeline
3. Groupee Search - it will search the groupee
4. Transfer Post - it will transfer the post from one bucket to other bucket of the same user
5. Groupee Transfer - it will transfer the groupee from one bucket to other bucket  irrespective of users

**Reply by Hanishkk** [reply]
All these features are nice, and user friendly but, is the design of app is user friendly too ?

Accordingly me, 
**Transfer post**, **Groupee**, and **Groupee transfer** should be added right now, rest can be added later with the updates.

At first let the users get to know about the app and its process, as I think so.

**Reply by egnimos** [reply]
yeah you are right!!, I have been working on UI, but the main thing is that, I want to make the app simple as possible, those who don't want to change their settings or any thing like that they can easily use the default settings without any hesitation, but making and adding all these features, makes the app little bit complicated, that's why I'm asking this question, to wait for the user to know this app, then release the next feature or add this features right now, in this current version

**Reply by egnimos** [reply]
> All these features are nice, and user friendly but, is the design of app is user friendly too ?
> 
> Accordingly me,
> **Transfer post**, **Groupee**, and **Groupee transfer** should be added right now, rest can be added later with the updates.
> 
> At first let the users get to know about the app and its process, as I think so.

yeah for design, you will get to see on sunday, I will add the **TEST APK** in this repo

**Comment by anonymousHa04** [comment]
What's groupee?

**Reply by egnimos** [reply]
it's just like a playlist, In groupee we usually add the list of post and provide the category to them, with some thumbnail, 
it help the user to group the post content into a single category and they can publish this groupee in the timeline.

**Reply by egnimos** [reply]
but the best part of groupee is that you can create the groupee and send it to any bucket

**Comment by egnimos** [comment]
Hey guy's, I have release the app, to download the **application**  you just have to go to the code section of this **repo** and check the **read me** or **discussion** there will be a link provided to the application, if you face any issue feel free to add it here along with the **screenshot** of the **error**

**Comment by egnimos** [comment]
post transfer support has been added, in same user, if you want to try please download it from the read me section

**Reply by Hanishkk** [reply]
I had one question, 
Can't you just choose two primary colours for the whole UI of the app?

**Reply by egnimos** [reply]
> I had one question, 
> Can't you just choose two primary colours for the whole UI of the app?
> 

Yup, I will
